alinka/widget/containers/main_body/footer/application.py

```diff
@@ -23,26 +23,16 @@ def __init__(self, parent: QWidget, visible: bool = False):
 
     @property
     def document_data(self):
-        return self.parent.parent.content_container.application_container.document_data
+        return self.parent.parent().content_container.application_container.document_data
 
     def validate_document_data(self) -> None:
         if not get_support_center_data():
             raise ValidationError()
 
     def print_documents(self) -> None:
-        try:
-            self.validate_document_data()
-            generate_and_save_decision(form_data=self.document_data, generate=True)
-        except Exception:
-            # to decide how we want handle this exception in separate ticket
-            # on handling exceptions
-            pass
+        self.validate_document_data()
+        generate_and_save_decision(form_data=self.document_data, generate=True)
```

Comment:
I'd better discuss it. Do you really want crash whole app in case of some invalid input, instead of informing user to update/change it?

Reply:
After check, it does't crash app, but just print error to console. So - it's ok for the time being. But let's find some better way to inform user about invalid data.

Reply:
There are two things from me here:
 * we don't crash app on exceptions - TBH I believe we should do! (but at the same time we should show actual exception to our users - otherwise it will be much more difficult to re-create the issue; I will work on such change in near future)
 * we should catch exceptions only when we know how to remedy it - purely doing
```python
try:
    ...
except Exception:
    pass
```
is one of the most malice we can do - that in most cases will move problem to other place and debugging that will require to untangle all these places looking for root cause.